howso/direct/client.py

```diff
@@ -2886,6 +2896,15 @@ def react(  # noqa: C901
                 " following values - ['min', 'max', 'most_similar',]"
             )
 
+        if details is not None and 'robust_computation' in details:
+            details['robust_influences'] = details['robust_computation']
+            details['robust_residuals'] = details['robust_computation']
+            del details['robust_computation']
+            warnings.warn(
+                'The detail "robust_computation" is deprecated and will be '
+                'removed in a future release. Please use "robust_residuals" '
+                'or "robust_influences" instead.', DeprecationWarning)
```

Comment:
Can use them together? Are their conditions where it makes sense to use both at the same time? If so, maybe use `..."robust_residuals" and/or "robust_influences"...`